### PR TITLE
Add max-width for pages with text in them, fix footer links

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/assets/styles/variables.scss [hunk 1]
```diff
@@ -49,3 +49,6 @@ $containerPaddingBottom: 4rem;
 // Line height
 $baseLineHeight: 1.2;
 $largeLineHeight: 1.6;
+
+// Maxwidth for text paragraphs
+$textParagraphMaxWidth: 50rem;
```
Comment (midori0507) [inline src/assets/styles/variables.scss]: https://i.dlpng.com/static/png/5480524_thumb.png
how come you come up with this idea??

File: src/domain/app/footer/Footer.tsx [hunk 2]
```diff
@@ -16,9 +17,15 @@ const Footer: FunctionComponent = () => {
             <p>{t('footer.copyrightText')}</p>
           </div>
           <div className={styles.links}>
-            <a href="/accessibility">{t('accessibilityStatement.title')}</a> •{' '}
-            <a href="/terms">{t('termsOfService.title')}</a> •{' '}
-            <a href={t('descriptionOfTheFile.url')}>
+            <Link to="/accessibility#start">
+              {t('accessibilityStatement.title')}
```
Comment (midori0507) [inline src/domain/app/footer/Footer.tsx]: The `{  }` is quite bad tho, can we replace it with padding / margin instead?

File: src/domain/accessibilityStatement/accessibilityStatement.module.scss [hunk 3]
```diff
@@ -3,6 +3,7 @@
 
 .accessibilityStatement {
   margin-bottom: $largeMargin;
+  max-width: $textParagraphMaxWidth;
```
Comment (midori0507) [inline src/domain/accessibilityStatement/accessibilityStatement.module.scss]: This look very duct tape

File: src/assets/styles/variables.scss (hunk 1 above)
Comment (badrange) [reply src/assets/styles/variables.scss]: Anniina. It is recommended max-line-length for text.

File: src/domain/app/footer/Footer.tsx (hunk 2 above)
Comment (badrange) [reply src/domain/app/footer/Footer.tsx]: It is a space character. We had it before and I'd like to keep it :-)

File: src/domain/accessibilityStatement/accessibilityStatement.module.scss (hunk 3 above)
Comment (badrange) [reply src/domain/accessibilityStatement/accessibilityStatement.module.scss]: Any idea on how to improve on this? The issue is that the lines should not be too long, it is difficult to read text which has very long lines.

File: src/domain/app/footer/Footer.tsx (hunk 2 above)
Comment (midori0507) [reply src/domain/app/footer/Footer.tsx]: alrigthy then

**Pull request merged**